static/js/stream_data.js

```diff
@@ -355,19 +355,27 @@ exports.get_color = function (stream_name) {
     return sub.color;
 };
 
-exports.in_home_view = function (stream_id) {
+exports.is_muted = function (stream_id) {
     var sub = exports.get_sub_by_id(stream_id);
-    return sub !== undefined && !sub.is_muted;
+    // Return true for undefined streams
+    if (sub === undefined) {
+        return true;
+    }
+    return sub.is_muted;
 };
 
-exports.name_in_home_view = function (stream_name) {
+exports.is_stream_name_muted = function (stream_name) {
```

Comment:
I think `is_stream_muted_by_name` would be more consistent with our other functions in this file.